CHANGELOG.md

```diff
@@ -3,6 +3,39 @@
 ## What's New
 
 * Revert dial error messages to what sdks are expecting. Add error codes so future sdks don't have to parse error text
+* Add router events
+* Allow filters with no predicate if sort or paging clauses are provided
+   * Ex: instead of `true limit 5` you could have just `limit 5`. Or instead of `true sort by name` you could have `sort by name`
+
+## Router events
+
+To enable:
+
+```
+events:
+  jsonLogger:
+    subscriptions:
+       - type: fabric.routers
```

Comment:
the extra space is here too - yaml gets cranky with that right?

Reply:
fixed. When I tested it, it had the extra space and it worked, but doesn't hurt to make it consistent